content/blog/2019-11-odbc-1-2-0/index.Rmarkdown

```diff
@@ -0,0 +1,108 @@
+---
+title: odbc 1.2.0
+author: Jim Hester
+date: '2019-11-27'
+slug: odbc-1-2-0
+categories:
+  - package
+tags:
+  - odbc
+photo:
+  url: https://unsplash.com/photos/40XgDxBfYXM
+  author: Photo by Jordan Harrison
+---
+
+odbc 1.2.0 is now on CRAN! The odbc package provides a DataBase Interface (DBI) to the Open DataBase Connectivity (ODBC) drivers. ODBC drivers exist for nearly all widely used databases, including [SQL
```

Comment:
Link somewhere in the intro paragraph to full change log (even though I know you have it at the end)? (Especially since the ODBC page isn't like regular pkgdown pages -- or at least the one I find when I look for it isn't).

Reply:
Yeah, the package does not have a pkgdown page, but we have `db.rstudio.com` which I definitely need to link to.